### DISCUSSION TITLE
Patch Release 2.8.1

### DISCUSSION THREAD

**Discussion by APickledWalrus** [SkriptLang/Skript#6384] [General]
# Skript 2.8.1

Skript 2.8.1 is here to resolve some of the most notable issues reported with 2.8.0. We will continue to assess stability and make fixes as necesary. As always, you can report any issues on our [issue tracker](https://github.com/SkriptLang/Skript/issues).

Happy Skripting!

### Additions
- [#6367](<https://github.com/SkriptLang/Skript/pull/6367>) Added support for experimental entities from 1.20.3, `breeze` and `wind charge`.

### Tweaks
- [#6357](<https://github.com/SkriptLang/Skript/pull/6357>) `armour` is now valid for the player armor change event.

### Bug Fixes
- [#6352](<https://github.com/SkriptLang/Skript/pull/6352>) Fixed a mapping issue that could cause an error on shutdown.
- [#6357](<https://github.com/SkriptLang/Skript/pull/6356>) Fixed an issue that caused `armor of %entities%` to be considered a single value.
- [#6358](<https://github.com/SkriptLang/Skript/pull/6358>) Fixed an issue that allowed invalid function definitions to parse successfully.
- [#6360](<https://github.com/SkriptLang/Skript/pull/6360>) Fixed parsing issues that caused valid code like `loop-value - 1` to error. Additionally, many improvements have been made to the `arithmetic` expression to greatly improve parsing and general stability.
- [#6374](<https://github.com/SkriptLang/Skript/pull/6374>) Fixed an issue that caused the `scripts` expression to return absolute paths for enabled scripts.
- [#6375](<https://github.com/SkriptLang/Skript/pull/6375>) Fixed an error that could occur when Skript attempted to interpret an unknown enumerator as a string.
- [#6378](<https://github.com/SkriptLang/Skript/pull/6378>) Fixed an invalid documentation link to the text tutorial.
- [#6383](<https://github.com/SkriptLang/Skript/pull/6383>) Fixed a syntax conflict that prevented the usage of the `vector from coordinates` expression.

[Click here to view the full list of commits made since 2.8.0](<https://github.com/SkriptLang/Skript/compare/2.8.0...2.8.1>)

### Help Us Test
We have an [official Discord community](https://discord.gg/ZPsZAg6ygu) for beta testing Skript's new features and releases.

### Thank You

Special thanks to the contributors whose work was included in this version:
- @APickledWalrus
- @AyhamAl-Ali
- @C0D3-M4513R
- @RamonJales
- @shaneBeee
- @sovdeeth

As always, if you encounter any issues or have some minor suggestions, please report them at https://github.com/SkriptLang/Skript/issues.
If you have any bigger ideas or input for the future of Skript, you can share those too at https://github.com/SkriptLang/Skript/discussions.

<hr /><em>This discussion was created from the release <a href='https://github.com/SkriptLang/Skript/releases/tag/2.8.1'>Patch Release 2.8.1</a>.</em>

**Comment by ghost** [comment]
ok, now this is cool
